pymeasure/instruments/lecroy/lecroyT3DSO1204.py

```diff
@@ -809,17 +841,20 @@ def _digitize(self, src, num_bytes=None):
             raise BufferError(f"read bytes ({len(binary_values)}) != requested bytes ({num_bytes})")
         return binary_values
 
-    def _header_sanity_checks(self, message):
+    def _header_footer_sanity_checks(self, message):
         """ Check that the header follows the predefined format.
         The format of the header is DAT2,#9XXXXXXX where XXXXXXX is the number of acquired
-        points and it is zero padded.
+        points, and it is zero padded.
+        Then check that the footer is present. The footer is a double line-carriage \n\n
         :param message: raw bytes received from the scope
-        :return: number of transmitted points as reported by the header """
+        :return: None """
```

Comment:
If you do not return anything, just drop that line.

Reply:
Done